### PR TITLE
docs: add chinese version readme.md

### LINKED ISSUE #286: [proposal] 增加一个中文翻译的 README
RT

### PR DESCRIPTION
**Pull request by ZiMengSheng**
### Ⅱ. Does this pull request fix one issue?
fixes #286 

### V. Checklist

- [X] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+[![Slack](https://badgen.net/badge/slack/join/4A154B?icon=slack&style=flat-square)](https://join.slack.com/t/koordinator-sh/shared_invite/zt-1756qoub4-Cn4~esfdlfAPsD7cwO2NzA)
+
+
+[English](./README.md) | 简体中文
+
+
+
+## 介绍
+
+Koordinator是基于QoS的调度系统，用于Kubernetes上多种工作负载的编排。它旨在提高工作负载的运行时效率和可靠性（包括延迟敏感型负载和批处理任务），简化资源相关的配置调整的复杂性，增加Pod部署密度以提高资源利用率。
```
[the remark this answers is not in the document]
Comment (ZiMengSheng) [reply README-zh_CN.md]: okay，重新润色了下

File: README-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- 采用应用画像机制（application profiling mechanism），支持超卖资源，以实现在满足 QoS 保障的前提下实现高资源利用率。
+- 细粒度的资源编排和隔离机制以提高工作负载（包括延迟敏感型负载和批处理任务）的效率。
+- 灵活的任务调度机制以支持特定领域（如大数据、AI、音频和视频）的工作负载。
+- 一套支持监控、故障排除、运维的工具集。
+
+## 快速开始
+
+你可以在[Koordinator website](https://koordinator.sh/docs)查看到完整的文档集。
+
+- 安装/升级Koordinator[最新版本](https://koordinator.sh/docs/installation)
```
Comment (saintube) [inline README-zh_CN.md:35]: 建议中英文连接的风格统一下，都空格，或者都不空格

**Review by eahydra — APPROVED** [review]
/lgtm

**Pull request merged**